Marlin/src/pins/lpc1769/pins_MKS_SGEN_L_V2.h

```diff
@@ -0,0 +1,365 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2020 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+#pragma once
+
+/**
+ * MKS SGen pin assignments
+ */
+
+#ifndef MCU_LPC1769
+  #error "Oops! Make sure you have the LPC1769 environment selected in your IDE."
+#endif
+
+#define BOARD_INFO_NAME   "MKS SGEN_L V2"
+#define BOARD_WEBSITE_URL "github.com/makerbase-mks"
+
+//
+// EEPROM, MKS SGEN_L V2.0 hardware has 4K EEPROM on the board
+//
+#if NO_EEPROM_SELECTED
+  //#define SDCARD_EEPROM_EMULATION
+  //#define I2C_EEPROM                            // AT24C32
+  #define FLASH_EEPROM_EMULATION
```

Comment:
Does the `FLASH_EEPROM_EMULATION` need to be enabled because the `I2C_EEPROM` doesn't yet work?

Reply:
Yes, AT24C32 does not work yet. So, use  FLASH_EEPROM_EMULATION

Reply:
The hardware is reserved, and I hope it can be the same as STM32 in the later period. User can select use FLASH_EEPROM_EMULATION or use AT24C32 for EEPROM data storage.

Reply:
So can we use the EEPROM after all? Is it a matter of working out some hardware kinks, or just testing it out?

Reply:
I2C_EEPROM function can be add in the future.